data/blog/open-source-in-valorizing-african-languages.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Afrim: Bridging the Gap with Open Source African Languages'
+date: 2024-03-28T10:30:00Z
+lastmod: '2024-03-28'
+tags: ['oss', 'african', 'linguistic']
+draft: true
+summary: 'The article discusses the potential of open source culture in Africa, particularly in relation to African languages. It emphasizes collaboration, inclusivity, and innovation. African languages, rich in diversity and history, are integral to the continent’s cultural heritage. Incorporating them into open source projects ensures linguistic diversity and accessibility. Initiatives like [afrim], an open source input method for African languages, play a crucial role. Challenges exist, such as the lack of standardized orthographies, but they also present opportunities for innovation. Organizations like [Geez.org] provide digital resources for specific languages. Overall, open source initiatives can drive growth and inclusivity, shaping the future of technology in Africa.'
+layout: PostLayout
+authors: ['pythonbrad']
+---
+<img src="https://tse2.mm.bing.net/th/id/OIG4.9PQw_eQ30kr5ETKCUems?pid=ImgGn" width="640" height="320" alt="intro">
```

Comment:
The CI build fails because you forgot to close the `<img>` tag.
Can you run `yarn build` on your local computer to ensure the project bundle succeed?